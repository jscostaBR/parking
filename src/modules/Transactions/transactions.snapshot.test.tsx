import * as React from 'react'
import { Transactions } from '.'
import renderer from 'react-test-renderer'

describe('render Resolutions', () => {
  it('render correctly with plate: AAA-1234 and type: payment', () => {
    const tree = renderer.create(
      <Transactions plate='AAA-1234' type='payment' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render correctly with plate: AAA-1234 and type: getOut', () => {
    const tree = renderer.create(
      <Transactions plate='AAA-1234' type='getOut' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
