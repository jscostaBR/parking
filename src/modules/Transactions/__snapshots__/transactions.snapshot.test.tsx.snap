// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Resolutions render correctly with plate: AAA-1234 and type: getOut 1`] = `
<div
  className="sc-bczRLJ kjaPWg"
>
  <p
    className="transaction-question"
  >
    Confirma 
    a sa√≠da
     da placa abaixo?
  </p>
  <p
    className="plate"
  >
    AAA-1234
  </p>
  <button
    className="sc-gsnTZi sc-hKMtZM iiaxUS kKxHSB"
    onClick={[Function]}
  >
    CONFIRMAR
  </button>
  <button
    className="go-back"
    onClick={[Function]}
  >
    Voltar
  </button>
</div>
`;

exports[`render Resolutions render correctly with plate: AAA-1234 and type: payment 1`] = `
<div
  className="sc-bczRLJ kjaPWg"
>
  <p
    className="transaction-question"
  >
    Confirma 
    o pagamento
     da placa abaixo?
  </p>
  <p
    className="plate"
  >
    AAA-1234
  </p>
  <button
    className="sc-gsnTZi sc-hKMtZM iiaxUS kKxHSB"
    onClick={[Function]}
  >
    CONFIRMAR
  </button>
  <button
    className="go-back"
    onClick={[Function]}
  >
    Voltar
  </button>
</div>
`;
